demo/scripts/controlsV2/sidePane/editorOptions/OptionState.ts

```diff
@@ -18,6 +18,7 @@ export interface NewPluginList {
     pasteOption: boolean;
     sampleEntity: boolean;
     markdown: boolean;
+    imageEditPlugin: boolean;
```

Comment:
Also remove the old one from LegacyPluginList. And now you can fully remove the interface LegacyPluginList and its UI.